content/en/developers/dogstatsd/_index.md

```diff
@@ -221,6 +221,90 @@ To gather custom metrics with [DogStatsD][1] with helm:
 [3]: https://github.com/containernetworking/cni
 [4]: https://kubernetes.io/docs/setup/independent/troubleshooting-kubeadm/#hostport-services-do-not-work
 {{% /tab %}}
+{{% tab "Operator" %}}
+
+To start collecting your StatsD metrics, you need to bind the DogStatsD port to a host port. You can also configure DogStatsD to use a [Unix domain socket][1].
+
+1. Add a `hostPort` to your `datadog-agent.yaml` manifest:
+
+    ```yaml
+    apiVersion: datadoghq.com/v1alpha1
+    kind: DatadogAgent
+    metadata:
+      name: datadog
+    spec:
+      agent:
+        config:
+          hostPort: 8125
+    ```
+
+     This enables your applications to send metrics with DogStatsD on port `8125` on whichever node they happen to be running. More operator configuration options here: https://docs.datadoghq.com/containers/kubernetes/operator_configuration/#all-configuration-options 
+
+     **Note**: `hostPort` functionality requires a networking provider that adheres to the [CNI specification][2], such as Calico, Canal, or Flannel. For more information, including a workaround for non-CNI network providers, see the Kubernetes documentation: [HostPort services do not work][3].
+
+2. Enable DogStatsD non local traffic to allow StatsD data collection, set `DD_DOGSTATSD_NON_LOCAL_TRAFFIC` to `true` in your `datadog-agent.yaml` manifest:
+
+    ```yaml
+    - name: DD_DOGSTATSD_NON_LOCAL_TRAFFIC
+      value: 'true'
+    ```
+
+     This allows collecting StatsD data from other containers than the one running the Agent.
+
+3. Apply the change:
+
+    ```shell
+    kubectl apply -f datadog-agent.yaml
+    ```
+
+**Warning**: The `hostPort` parameter opens a port on your host. Make sure your firewall only allows access from your applications or trusted sources. If your network plugin doesn't support `hostPorts`, so add `hostNetwork: true` in your Agent pod specifications. This shares the network namespace of your host with the Datadog Agent. It also means that all ports opened on the container are opened on the host. If a port is used both on the host and in your container, they conflict (since they share the same network namespace) and the pod does not start. Some Kubernetes installations do not allow this.
```

Comment:
```suggestion
**Warning**: The `hostPort` parameter opens a port on your host. Make sure your firewall allows access only from your applications or trusted sources. If your network plugin doesn't support `hostPorts`, add `hostNetwork: true` in your Agent pod specifications. This shares the network namespace of your host with the Datadog Agent. It also means that all ports opened on the container are opened on the host. If a port is used both on the host and in your container, they conflict (since they share the same network namespace) and the pod does not start. Some Kubernetes installations do not allow this.
```